docs/buildah-manifest.md

```diff
@@ -0,0 +1,30 @@
+# buildah-manifest "1" "September 2019" "buildah"
+
+## NAME
+buildah-manifest - Create and manipulate manifest lists and image indexes.
+
+## SYNOPSIS
+buildah manifest COMMAND [OPTIONS] [ARG...]
+
+## DESCRIPTION
+The `buildah manifest` command provides subcommands which can be used to:
+
+    * Create a working Docker manifest list or OCI image index.
+    * Add an entry to a manifest list or image index for a specified image.
+    * Add or update information about an entry in a manifest list or image index.
+    * Delete a working container or an image.
+    * Push a manifest list or image index to a registry or other location.
+
+## SUBCOMMANDS
+
+| Command  | Man Page                                                     | Description                                                                 |
+| -------  | ------------------------------------------------------------ | --------------------------------------------------------------------------- |
+| create   | [buildah-manifest-create(1)](buildah-manifest-create.md)     | Create a manifest list or image index.                                      |
+| add      | [buildah-manifest-add(1)](buildah-manifest-add.md)           | Add an image to a manifest list or image index.                             |
+| remove   | [buildah-manifest-remove(1)](buildah-manifest-remove.md)     | Remove an image from a manifest list or image index.                        |
```

Comment:
could you move this remove line down below push please?

Reply:
Sure, but just so we're on the same page, this doesn't remove the list -- that's something that `buildah rmi` would handle.